app.js

```diff
@@ -86,6 +87,7 @@ app.use(cookieParser());
 
 app.use('/', index);
 app.use('/chart_test', chartTest);
+app.use('/devicereg', deviceReg);
```

Comment:
Can we route this to `/devices` instead?

Then the new action will be `GET /devices/new` and the create action will be `POST /devices` .

I'ts just a convention. I'm following it for chorals right now, so it would be sweet to be consistent